frontend/src/components/Plan/PlanIdeaList.module.css

```diff
@@ -0,0 +1,36 @@
+.ideaList {
+  margin: 1rem 1rem;
+  background-color: #F5f5f7;
```

Comment:
nit; given we have color variables being used, it's generally good to define colors rather than put in hex codes everywhere